monitoring-samples/stream-apps/README.md

```diff
@@ -1,37 +1,41 @@
 # Stream Apps Monitoring
 
-Ensure that the POM contains:
+Add `org.springframework.cloud:spring-cloud-stream` and `org.springframework.cloud:spring-cloud-starter-stream-rabbit` to your Spring Boot application 
+POM to turn it into Stream Cloud Stream Application:
 
 ```xml
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-starter-stream-rabbit</artifactId>
-		</dependency>
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-stream</artifactId>
+    </dependency>
+    
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-starter-stream-rabbit</artifactId>
+    </dependency>
+```
 
+To replace Rabbit by Kafka binder use the `org.springframework.cloud:spring-cloud-starter-stream-kafka` dependency instead instead.
 
-		<dependency>
-			<groupId>org.springframework.cloud.stream.app</groupId>
-			<artifactId>app-starters-micrometer-common</artifactId>
-			<version>2.1.2.RELEASE</version>
-		</dependency>
-		<dependency>
-			<groupId>io.micrometer</groupId>
-			<artifactId>micrometer-registry-prometheus</artifactId>
-		</dependency>
+To enable SCDF Prometheus Proxy monitoring for your application you would need to add the following additional additional dependencies: 
 
-		<dependency>
-			<groupId>io.micrometer.prometheus</groupId>
-			<artifactId>prometheus-rsocket-spring</artifactId>
-			<version>0.9.0</version>
-		</dependency>
-		...
-	</dependencies>
+```xml
+    <dependency>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>app-starters-micrometer-common</artifactId>
+        <version>2.1.2.RELEASE</version>
+    </dependency>
+    <dependency>
+        <groupId>io.micrometer</groupId>
+        <artifactId>micrometer-registry-prometheus</artifactId>
+    </dependency>    
+    <dependency>
+        <groupId>io.micrometer.prometheus</groupId>
+        <artifactId>prometheus-rsocket-spring</artifactId>
+        <version>0.9.0</version>
+    </dependency>
 ```
 
-For Kafka binder replace `spring-cloud-starter-stream-rabbit` by `spring-cloud-starter-stream-kafka` instead
+The `micrometer-registry-prometheus` amd `prometheus-rsocket-spring` activate the Micrometer Prometheus support for RSocket Proxy meter registry.
+The `app-starters-micrometer-common` would inject some common metrics Tags such as `stream.name`, `stream.app.label`, `application.type` and `application.guid`.
```

Comment:
Maybe we could describe it in the context of SCDF. Or even in the context of the OOTB Grafana Dashboard that we ship, and why these tags are important for the PromQL.

Reply:
the tags set by the app-starters-micrometer-common a generic for all ~15 TSDBs supported by micrometer  including Prometheus and Influx DB.  We use them for the Influx Dashboards as well. They can be used to aggregate metrics by stream, apps and app instance. For example we used them to implement the Rate-delta alert in the scaling demo. 
Tags are briefly explained in the microsite [here](https://dataflow.spring.io/docs/2.3.0.SNAPSHOT/feature-guides/streams/monitoring/) (btw i need to update the diagrams there as well)
Also imo, the focus of this README is on how to make it work even if i don't understand it. E.g. the goal is to plug a custom Stream App in the existing SCDF monitoring architecture. The detailed explanation of the architecture should be in the microsite?